ui/src/services/platformlibrary/k8s.js

```diff
@@ -52,3 +54,184 @@ export const getVolumesCountQuery = (
     enabled: token ? true : false,
   };
 };
+
+// the queries for the metrics
+// TODO: we may want to merge useStartingTimeStamp() and useMetricsTimeSpan(), so the props related to timespan will be returned in one
+export const getCPUUsageQuery = (
+  instanceIP: string,
+  timespanProps: {
```

Comment:
You can extract this timespanProps type so that it can be reused by all the queries

Reply:
sure. And I am thinking about prehaps I shouldn't put those queries in k8s.js
I will create another folder called `metrics.js`